docs/algorithms/sig/picnic.md

```diff
@@ -9,6 +9,9 @@
   - **Source**: https://github.com/pq-crystals/kyber/commit/faf5c3fe33e0b61c7c8a7888dd862bf5def17ad2 with copy_from_upstream patches
   - **Implementation license (SPDX-Identifier)**: CC0-1.0
 
+## Test limitation
```

Comment:
I guess this note will get wiped out when the Markdown documentation is regenerated, but hopefully the main problem will be solved by then.